webxdc.js

```diff
@@ -1,125 +1,82 @@
 // debug friend: document.writeln(JSON.stringify(value));
-window.xdcStorage = (() => {
+window.webxdc = (() => {
+    var updateListener = () => {};
     var updatesKey = "__xdcUpdatesKey__";
-    var fakeStorage = {
-        _data: {},
-
-        setItem: function (id, val) {
-            return this._data[id] = String(val);
-        },
-
-        getItem: function (id) {
-            return this._data.hasOwnProperty(id) ? this._data[id] : undefined;
-        },
-
-        removeItem: function (id) {
-            return delete this._data[id];
-        },
-
-        clear: function () {
-            return this._data = {};
-        }
-    };
-    var localStorageSupported = () => {
-        var testKey = "__xdcTestKey__";
-        try {
-            var storage = window.localStorage;
-            storage.setItem(testKey, "1");
-            storage.removeItem(testKey);
-            return true;
-        } catch (error) {
-            return false;
+    window.addEventListener('storage', (event) => {
```

Comment:
why is that here and not on top-level as the other added event-listener? if there are no special reasons, i'd keep similar things together.

Reply:
because updatesKey is secret/private to the webxdc object, I could rework it and use a global const maybe renaming it to xdcUpdatesKey to avoid possible collisions with app's code

Reply:
> because updatesKey is secret/private to the webxdc object, I could rework it and use a global const maybe renaming it to xdcUpdatesKey to avoid possible collisions with app's code

Then there's a good reason for it. Keep it as it is for now :smile:

Reply:
yip, that is a good reason, just keep it, i was just asking :)